tests/sdfg/race_conditions_check.py

```diff
@@ -0,0 +1,34 @@
+import dace
+import numpy as np
+import warnings
+warnings.filterwarnings("error")
```

Comment:
with pytest, you shouldn't use this mode, you can use a `with` statement that expects a warning